MetaMask/metamask-extension
# [FLASK] Fix an issue with installing and updating snaps with 0 permissions

## Explanation
Fixes an issue with installing snaps that didn't request any permissions.

## Manual Testing Steps

1. Create a snap that requests 0 permissions
2. Try to install the snap
3. See that the install prompt shows correctly
4. Verify that you can install a snap and reject installation of a snap
